### PR TITLE
Fix error in  when no credential store file exists

### PR DESCRIPTION
**Pull request by KingEmet**
Manifests as "could not retrieve 3p credentials" during `docker build`

Change-Id: I981d1a4dc48862887b0063e7b47f4a5349d6b5b7
Signed-off-by: Jake Sanders <jsand@google.com>

### CODE REVIEW

File: credhelper/helper_integration_test.go [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+	"github.com/GoogleCloudPlatform/docker-credential-gcr/store"
+	"github.com/docker/docker-credential-helpers/credentials"
+)
+
+var expectedGcrHosts = [...]string{
+	"gcr.io",
+	"us.gcr.io",
+	"eu.gcr.io",
+	"asia.gcr.io",
+	"b.gcr.io",
```
Comment (mattmoor) [inline credhelper/helper_integration_test.go]: May as well remove the bucket versions

File: credhelper/helper_integration_test.go [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+	"bucket.gcr.io",
+	"appengine.gcr.io",
+	"gcr.kubernetes.io",
+	"beta.gcr.io",
```
Comment (mattmoor) [inline credhelper/helper_integration_test.go]: I'd remove this too.

File: credhelper/helper_integration_test.go (hunk 1 above)
Comment (KingEmet) [inline credhelper/helper_integration_test.go]: ack

File: credhelper/helper_integration_test.go (hunk 2 above)
Comment (KingEmet) [inline credhelper/helper_integration_test.go]: ack

**Comment by KingEmet** [comment]
Will modify the default registries in a separate change

**Pull request merged**